### PR TITLE
[add] home page UI

### PR DESCRIPTION
**Pull request by Digital-Human-BK**
Hey Stefan. I added many things again, but they won't work without each other.

### CODE REVIEW

File: src/common/Toast/Toast.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import MuiAlert, { AlertProps, AlertColor } from '@mui/material/Alert';
+
+import { useAppDispatch } from '../../configureStore';
+// import { resetError } from '../../../store/auth-slice';
+
+const Alert = React.forwardRef<HTMLDivElement, AlertProps>(function Alert(
+  props,
+  ref
+) {
+  // eslint-disable-next-line react/jsx-props-no-spreading
```
Comment (stefanpetkov90) [inline src/common/Toast/Toast.tsx:13]: add the following to .eslintrc.json "rules" section :

"react/jsx-props-no-spreading": "off"

**Review by stefanpetkov90 — CHANGES_REQUESTED** [review]

File: src/common/Toast/Toast.tsx (hunk 1 above)
Comment (Digital-Human-BK) [reply src/common/Toast/Toast.tsx]: added successfully

File: src/common/LinkComponent/LinkComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+type LinkComponentProps = {
+  children: string;
+  path: string;
+  styles: object;
+};
+
+function LinkComponent({ children, path, styles }: LinkComponentProps) {
+  const navigate = useNavigate();
+  return (
+    // eslint-disable-next-line jsx-a11y/anchor-is-valid
```
[the remark this answers is not in the document]
Comment (stefanpetkov90) [reply src/common/LinkComponent/LinkComponent.tsx:12]: In that case set it to "off" and disable the check

[the remark this answers is not in the document]
Comment (Digital-Human-BK) [reply src/common/LinkComponent/LinkComponent.tsx]: done

**Review by stefanpetkov90 — APPROVED** [review]

**Pull request merged**